hermes-consumers/src/main/java/pl/allegro/tech/hermes/consumers/consumer/ConsumerMessageSender.java

```diff
@@ -131,6 +136,9 @@ private int calculateMessageDelay(long publishingMessageTimestamp) {
      */
     private void sendMessage(final Message message) {
         loadRecorder.recordSingleOperation();
+        HermesTimerContext acquireTimer = rateLimiterAcquireTimer.time();
+        rateLimiter.acquire();
```

Comment:
**question:** Does only `ConsumerMessageSender` use the `send` method from `ResilientMessageSender`?

Reply:
Actually `ConsumerMessageSender` use the send method from `JettyBroadCastMessageSender` or `SingleRecipientMessageSenderAdapter` depends on `MessageSender` implementation provided by factory. True is, that those two classes which i have mentioned are the only usages of `ResilientMessageSender.send()`